scripts/deploy_assisted_service.sh

```diff
@@ -135,7 +135,7 @@ else
         minikube image rm "${SERVICE}" || true
     fi
 
-    skipper run src/update_assisted_service_cm.py
+    skipper run python3 src/update_assisted_service_cm.py
```

Comment:
now this change should be redundant I think

Reply:
On most other places we call python3 explicitly, I think we should move them all to exeute it using `skipper run python3 ...`

Reply:
so why having the shebang line if now using it?